docs/reference/settings/ml-settings.asciidoc

```diff
@@ -77,40 +78,41 @@ adding an `inference` processor to a pipeline is disallowed. Defaults to `50`.
 
 `xpack.ml.max_machine_memory_percent`::
 (<<cluster-update-settings,Dynamic>>) The maximum percentage of the machine's
-memory that {ml} may use for running analytics processes. (These processes are
-separate to the {es} JVM.) Defaults to `30` percent. The limit is based on the
-total memory of the machine, not current free memory. Jobs are not allocated to
-a node if doing so would cause the estimated memory use of {ml} jobs to exceed
-the limit. When the {operator-feature} is enabled, this setting can be updated
-only by operator users.
+memory that {ml} may use for running analytics processes. These processes are
+separate to the {es} JVM. The limit is based on the total memory of the machine, 
+not current free memory. Jobs are not allocated to a node if doing so would
+cause the estimated memory use of {ml} jobs to exceed the limit. When the
+{operator-feature} is enabled, this setting can be updated only by operator
+users. The minimum value is `5`; the maximum value is `200`. Defaults to `30`.
 
 `xpack.ml.max_model_memory_limit`::
 (<<cluster-update-settings,Dynamic>>) The maximum `model_memory_limit` property
 value that can be set for any job on this node. If you try to create a job with
 a `model_memory_limit` property value that is greater than this setting value,
 an error occurs. Existing jobs are not affected when you update this setting.
+If this setting is `0` or unset, the maximum `model_memory_limit` value is `1gb`.
```

Comment:
```suggestion
If this setting is `0` or unset, there is no maximum `model_memory_limit` (which makes it possible to create jobs that cannot be assigned to any available nodes).
```

Reply:
This is true when job is created with `AnalysisLimits. model_memory_limit` set. If unset then then `AnalysisLimits. model_memory_limit` takes on the value of `xpack.ml.max_model_memory_limit` if set else 1GB. 

Suggestion:
> If this setting is `0` or unset, there is no maximum to the `model_memory_limit` setting for AD and DFA jobs (which makes it possible to create jobs that cannot be assigned to any available nodes).


Then in the [PUT Job docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-put-job.html) section about `model_memory_limit` we could change 

 > The default value for jobs created in version 6.1 and later is 1024mb.

to 

> The default value for jobs created in version 6.1 and later is `xpack.ml.max_model_memory_limit`. If `xpack.ml.max_model_memory_limit` is `0` or unset `model_memory_limit` is set to `1024mb`.

Reply:
> ... which makes it possible to create jobs that cannot be assigned to any available nodes.

Since the first sentence of this setting description says it affects "... any job on this node", should the new clarification also be limited to this node? For example: "...which makes it possible to create jobs that cannot run on this node if it doesn't meet those memory requirements".?